src/reaction_network.jl

```diff
@@ -186,36 +145,74 @@ emptyrn = @reaction_network empty
 # an empty network with random generated name
 emptyrn = @reaction_network
 ```
+
+ReactionSystems generated through `@reaction_network` are complete.
 """
+# macro reaction_network(args...)
+#     return :(complete(@network_component $(args... )))
+# end
```

Comment:
Ideally, we'd only want to implement something like this macro, where `@reaction_network` calls `@reaction_component` (which does the actual work).

Unfortunately, this implementation does not work with interpolation. E.g.
```julia
name = :NAME
rn = @reaction_network $NAM begin
    d, X --> 0
end

k = 1
rn = @reaction_network begin
    d + 1 + $k, X --> 0
end
``` 
gives `name` and `k` `not defined`, respectively. Figured I'd let you have a look, since you know interpolation well and might have an easy solution.

Reply:
removed